src/util/cmdlineargs.cpp

```diff
@@ -177,7 +177,7 @@ bool CmdlineArgs::parse(const QStringList& arguments, CmdlineArgs::ParseMode mod
                             : QString());
     QCommandLineOption controllerDebugDeprecated(
             QStringList({QStringLiteral("controllerDebug"),
-                    QStringLiteral("midiDebug")}));
+                    QStringLiteral("controllerDebug")}));
```

Comment:
This change seems wrong.

Reply:
good catch, classic search+replace error